CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.6-M1
```

Comment:
please make the version as 0.2.6 only and remove '-M1' from it.

Reply:
Thanks for the review comment @agg3fe  . I have incorporated the review comment.